org.eclipse.lemminx/src/test/java/org/eclipse/lemminx/utils/ProjectUtils.java

```diff
@@ -25,7 +25,7 @@
 	 * @return the current lemminx project directory
 	 */
 	public static Path getProjectDirectory() {
-		String currPath = new File(ProjectUtils.class.getClassLoader().getResource("").getPath()).toString();
+		String currPath = new File(ProjectUtils.class.getClassLoader().getResource("org/eclipse/lemminx/XMLAssert.class").getPath()).toString();
```

Comment:
add a comment in org/eclipse/lemminx/XMLAssert.java to remind the next maintainer that refactoring that class will require updating ProjectUtils too

Reply:
@fbricon I generate this XMLAssert.class, so after a refactoring, it will continue to work.